src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java

```diff
@@ -49,9 +44,13 @@
 import appeng.tile.misc.TileInterface;
 import appeng.util.InventoryAdaptor;
 import appeng.util.Platform;
-import appeng.util.inv.AdaptorIInventory;
-import appeng.util.inv.AdaptorPlayerHand;
-import appeng.util.inv.WrapperInvSlot;
+import appeng.util.inv.AdaptorItemHandler;
+import net.minecraft.entity.player.EntityPlayerMP;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'net.minecraft.entity.player.EntityPlayerMP' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)